GridKit/Model/PhasorDynamics/Branch/BranchData.hpp

```diff
@@ -31,18 +32,38 @@ namespace GridKit
     /// Variables able to be monitored for a branch
     enum class BranchMonitorableVariables
     {
-      ir1,
-      ii1,
-      im1,
-      p1,
-      q1,
-      ir2,
-      ii2,
-      im2,
-      p2,
-      q2,
+      IR1,
+      II1,
+      IM1,
+      P1,
+      Q1,
+      IR2,
+      II2,
+      IM2,
+      P2,
+      Q2,
+      SIZE
     };
 
+    /**
```

Comment:
just calling this out: if we go along with permitting magic enum to be used in this context, we'd have to undo the guideline changes and then we would be able to get rid of this convenience function

Reply:
@superwhiskers, could you explain what you mean by this?